lib/ansible/utils/module_docs_fragments/eos.py

```diff
@@ -24,6 +24,11 @@ class ModuleDocFragment(object):
 options:
   authorize:
     description:
+      - B(Deprecated)
+      - "Starting with Ansible 2.5 we recommend using C(connection: network_cli) and C(become: yes)."
+      - This option is only required if you are using eAPI.
+      - For more information please see the L(EOS Platform Options guide, ../network/user_guide/platform_eos.html).
```

Comment:
L(EOS Platform Options guide, ../network/user_guide/platform_eos.html#using-cli-in-ansible-2-5)